tests/aderyn.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "use_detectors": "push-zero-opcode,useless-public-function",
```

Comment:
Could we make this an array instead of a string with comma-delimited input?

```json
"use_detectors": [
   "push-zero-opcode",
   "useless-public-function"
]
```

Reply:
Ah okay! I like that too : ) but more to type tho .  People will be more used to this because slither :P What do you think ?